ansible/example_site_secrets.yml

```diff
@@ -115,7 +119,7 @@ project_app_root: '{{project_user_home }}/{{ project_name }}'
 # Username for chroot SFTP user
 sftp_user: "upload"
 # Username that should be the primary owner of the chroot SFTP directory area
-sftp_chroot_owner: "{{ project_user }}"
+sftp_chroot_owner: "{{ project_run_user }}"
```

Comment:
You'll have to define better what is considered a breaking change.  Here, I'm just changing the value of an `example_site_secrets.yml` variable, not changing the name or deleting an existing variable.  However, the change was introduced because of the change in semantics due to the new run/owner users that were introduced in this change.  Those are definitely breaking semantics compared to the previous version of `master`.

So, having said that, I'm happy for us to bump the semantic version and tag a new release every time we have non-comment changes to `example_site_secrets.yml`.